coral-component-masonry/examples/index.html

```diff
@@ -283,25 +300,32 @@ <h2 class="coral-Heading--M">Usage notes</h2>
             }
 
             function createImageItem() {
+              var id = getUID();
+              var contentId = id + '-content';
               var width = randomSize();
               var height = randomSize();
               var url = 'http://via.placeholder.com/' + width + 'x' + height;
 
               var item = document.createElement('coral-masonry-item');
+              item.id = id;
+              item.setAttribute('aria-labelledby', contentId);
               item.classList.add('coral-Well');
-              item.content.innerHTML = '<img alt="" src="' + url + '" style="width: ' + (width / 2) + 'px; height: ' + (height / 2) + 'px;">';
+              item.content.innerHTML = '<img id="' + contentId + '" alt="' + width + ' × ' + height + '" src="' + url + '" style="width: ' + (width / 2) + 'px; height: ' + (height / 2) + 'px;">';
 
               return item;
             }
 
             function createTextItem() {
+              var id = getUID();
```

Comment:
Please use commons.getUID to get unique id. https://github.com/adobe/coral-spectrum/blob/master/coral-utils/src/scripts/Commons.js#L555